repository/gridftp/globus5/authz/tests/unit/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# Each test listed in Alphabetical order
+foreach(PROG
+    test_getVersion
```

Comment:
So is this being used as a string? Is this a variable I'm missing somewhere? I see the filename, but I'm not sure how the `foreach` interprets this list when it's just the unquoted filename.

Reply:
It just treats it as a string.